### ISSUE TITLE
goldendict auto adds gxxxxxx in front of html id name which makes JavaScript failed

### ISSUE THREAD

**Issue by WanderMax** [goldendict/goldendict#902]
I find that goldendict auto adds some strings in front of html id name, and it makes the JavaScript function failed.
You can as below, the highlighted part makes JavaScript that runs with id name failed.
![auto_add](https://user-images.githubusercontent.com/15909056/31338068-7fdcfd0c-ad30-11e7-89f7-0361a6840e14.png)

**Comment by Abs62** [comment]
It is not bug, it is feature.

**Comment by WanderMax** [comment]
how can I solve the problem, make id name not changed?

**Comment by Abs62** [comment]
I think it is simpler to change JS script.

**Comment by WanderMax** [comment]
terrible, only the id names are different. any method to disable this feature in specified instance?

**Comment by Abs62** [comment]
No such method. GD add dictionary/article ID to elements IDs to create unique anchors because same IDs can be presented in other dictionary/article in the current dictionary group.  
What dictionary have such problem?

**Comment by WanderMax** [comment]
I grab the contents from Bing dictionary and make the offline dictionary.
The JS script makes use of the id name to switch between the tabs.

**Comment by Abs62** [comment]
Article in GD is not same thing as page on site. In general case page in GD can contain several articles with same (originally) IDs, for example from different versions of same dictionary. Unique modifiers give you possibility to not miss while search of the target element. Just modify JS script to handle modified IDs correctly.

**Comment by WanderMax** [comment]
@Abs62 can I  get the dictionary and article ID via js script?

**Comment by Abs62** [comment]
Yes, from modified elements ID. Up to first '_' - dictionary ID, from first to second '_' - article one.  
You can take original ID via JS string.substr() method, for example. Or just use string.lastIndexOf() istead of direct string compare.

**Comment by WanderMax** [comment]
@Abs62 still very terrible for these JS scripts handle with id name!
